channeldb/meta.go

```diff
@@ -45,10 +55,10 @@ func (d *DB) FetchMeta(tx kvdb.RTx) (*Meta, error) {
 	return meta, nil
 }
 
-// fetchMeta is an internal helper function used in order to allow callers to
-// re-use a database transaction. See the publicly exported FetchMeta method
-// for more information.
-func fetchMeta(meta *Meta, tx kvdb.RTx) error {
+// FetchMeta is a helper function used in order to allow callers to re-use a
+// database transaction. See the publicly exported FetchMeta method for more
```

Comment:
I think this sentence can now be deleted `See the publicly exported FetchMeta...`